datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/dashboards.py

```diff
@@ -81,11 +79,23 @@ def dashboards():
                 echo_failure(' FAILED')
                 for display_func, message in display_queue:
                     display_func(message)
+                display_queue = []
             else:
                 ok_checks += 1
 
+            if file_old_format:
+                old_payloads += 1
+                # Display detailed info if file is using the old screen/time payloads
+                echo_info(f'{check_name}... ', nl=False)
+                echo_warning(' WARNING')
+                for display_func, message in display_queue:
+                    display_func(message)
+                display_queue = []
+
     if ok_checks:
         echo_success(f"{ok_checks} valid files")
+    if old_payloads:
+        echo_warning(f"{old_payloads} using the old screen/time payloads and should be migrated")
```

Comment:
Should we hold off on this warning until we've have active plans to migrate?

The output is really loud since all files are using old endpoint payloads:

```

    /Users/christine.chen/dev/integrations-core/scylla/assets/dashboards/overview.json is not using the new /dashboard payload format.
sidekiq...  WARNING
    /Users/christine.chen/dev/integrations-core/sidekiq/assets/dashboards/overview.json is not using the new /dashboard payload format.
systemd...  WARNING
    /Users/christine.chen/dev/integrations-core/systemd/assets/dashboards/overview.json is not using the new /dashboard payload format.
tls...  WARNING
    /Users/christine.chen/dev/integrations-core/tls/assets/dashboards/overview.json is not using the new /dashboard payload format.
vault...  WARNING
    /Users/christine.chen/dev/integrations-core/vault/assets/dashboards/vault_overview.json is not using the new /dashboard payload format.
vertica...  WARNING
    /Users/christine.chen/dev/integrations-core/vertica/assets/dashboards/overview.json is not using the new /dashboard payload format.
vsphere...  WARNING
    /Users/christine.chen/dev/integrations-core/vsphere/assets/dashboards/vsphere_overview.json is not using the new /dashboard payload format.
44 valid files
44 using the old screen/time payloads and should be migrated
```

Reply:
We're currently working on migrating these actively (see #integration-dashboard-migration). It would also be great if other teams could pitch in and migrate these; the noise outweights the time lost between getting these old formats to generate the boards.